app/controllers/parties_controller.rb

```diff
@@ -38,20 +37,16 @@ def letters
   def members_letters
     letter = params[:letter]
     party_id = params[:party_id]
-    data_party = Parliament::Request.new.parties(party_id).get
     data_members = Parliament::Request.new.parties(party_id).members(letter).get
-
-    @party = data_party.first
+    @party = data_members.filter('http://id.ukpds.org/schema/Party').first
```

Comment:
These filter lines should be consolidated into a single filter call - this is currently 2N